src/Umbraco.Web.UI.Client/src/common/resources/media.resource.js

```diff
@@ -550,36 +550,6 @@ function mediaResource($q, $http, umbDataFormatter, umbRequestHelper) {
                         "Search",
                         args)),
                 'Failed to retrieve media items for search: ' + query);
-        },
-
-        getPagedReferences: function (id, options) {
```

Comment:
This is a breaking change

Reply:
I think it's fine for a breaking change, i don't believe that this method is documented so people probably haven't used it but would mean this needs to be marked as a breaking change.

Reply:
I could put it back and call the new api method or the new resource. So it would not break, but we can mark the method as obsolete

Reply:
I put the method back so the change is non-breaking. The method calls the new resource. Also marked as deprecated for the docs